concent_api/utils/constants.py

```diff
@@ -57,7 +57,11 @@ class ErrorCode(enum.Enum):
     QUEUE_WRONG_STATE                                                   = 'queue.wrong_state'
     REQUEST_BODY_NOT_EMPTY                                              = 'request_body.not_empty'
     SUBTASK_DUPLICATE_REQUEST                                           = 'subtask.duplicate_request'
+    VERIFIIER_COMPUTING_SSIM_FAILED                                     = 'verifier.computing_ssim_failed'
     VERIFIIER_FILE_DOWNLOAD_FAILED                                      = 'verifier.file_download_failed'
+    VERIFIIER_LOADING_FILES_INTO_MEMORY_FAILED                          = 'verifier.loading_files_into_memory_failed'
```

Comment:
I am mot sure if we need to distinguish between these two:
`VERIFIIER_LOADING_FILES_INTO_MEMORY_FAILED` vs `VERIFIIER_LOADING_FILES_WITH_OPENCV_FAILED`

Reply:
You have double `I` here.

Reply:
Right, I will get rid of second one.

Reply:
:+1: done